templates/components/cart/totals.html

```diff
@@ -71,25 +71,27 @@
             {{> components/cart/coupon-input}}
         </li>
     {{/if}}
-    {{#each cart.gift_certificates}}
-        <li class="cart-total">
-            <div class="cart-total-label">
-                <strong>{{lang 'cart.gift_certificates.code_label' code=code}}</strong>
-                <span>
-                    {{remaining.formatted}} | <a href="{{remove_url}}">{{lang 'cart.gift_certificates.remove'}}</a>
-                </span>
-            </div>
-            <div class="cart-total-value">
-                {{used.formatted}}
-            </div>
-        </li>
-    {{/each}}
-    <li class="cart-total">
-        <div class="cart-total-label">
-            <strong>{{lang 'cart.gift_certificates.gift_certificate'}}:</strong>
-        </div>
-        {{> components/cart/gift-certificate-input}}
-    </li>
+    {{#if settings.gift_certificates_enabled}}
```

Comment:
You don't need this flag because when gift certificates are disabled `cart.gift_certificates` should be empty.

if `cart.gift_certificates` is not empty is because a bcapp bug.